server/src/internalClusterTest/java/org/elasticsearch/index/shard/SearchIdleIT.java

```diff
@@ -182,4 +200,27 @@ private void ensureNoPendingScheduledRefresh(ThreadPool threadPool) {
         barrier.arriveAndAwaitAdvance();
     }
 
+    public void testSearchIdleStats() throws InterruptedException {
+        int searchIdleAfter = randomIntBetween(2, 5);
+        final String indexName = randomAlphaOfLength(5).toLowerCase(Locale.ROOT);
+        client().admin()
+            .indices()
+            .prepareCreate(indexName)
+            .setSettings(
+                Settings.builder()
+                    .put(IndexSettings.INDEX_SEARCH_IDLE_AFTER.getKey(), searchIdleAfter + "s")
```

Comment:
Maybe reduce this idle after to something like 100ms here? Otherwise it increases the time to run this test significantly.